olp/utils.py

```diff
@@ -105,6 +133,7 @@ def patch_models():
 
     setattr(User, "assign_perm", assign_perm)
     setattr(User, "remove_perm", remove_perm)
+    setattr(User, "remove_all_permissions", remove_all_permissions)
```

Comment:
Right now we are only monkey patching the `User` model so we add onto existing methods. I'd rather not add a `remove_all_permissions` method just to that model, and instead require that the helper method is used directly.